src/directories.ts

```diff
@@ -0,0 +1,52 @@
+import {window, commands, StatusBarAlignment, StatusBarItem, workspace, extensions} from 'vscode';
+import path = require('path');
+
+let fs = require('fs');
+
+
+var folders: Array<string> = ["Scenes", "Prefabs", "Scripts", "Materials", "Audio"];
+
+export function GenerateOrganizationFolders(path: string)
+{
+    var generationFolders;
+    generationFolders = setDefaultOrCustomFolders();
+    for(var j = 0; j <generationFolders.length; j++){
+        var dir = path + generationFolders[j];
+        fs.mkdir(dir, function (err) {
```

Comment:
What happens if folder already exists? No error messages to user? Can't can't you check EEXIST or ENOENT?